packages/console/src/components/MultilineInput/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as textButtonStyles from '@/components/TextButton/index.module.scss';
+import Minus from '@/icons/Minus';
+
+import IconButton from '../IconButton';
+import TextInput from '../TextInput';
+import * as styles from './index.module.scss';
+
+type Props = {
+  value: string[];
+  onChange: (value: string[]) => void;
+};
+
+const MultilineInput = ({ value, onChange }: Props) => {
+  const { t } = useTranslation(undefined, {
+    keyPrefix: 'general',
+  });
+
+  const fields = useMemo(() => {
+    if (value.length === 0) {
+      return [''];
+    }
+
+    return value;
+  }, [value]);
+
+  const handleAdd = () => {
+    onChange([...value, '']);
+  };
+
+  const handleRemove = (index: number) => {
+    onChange(fields.filter((_, i) => i !== index));
+  };
+
+  const handleInputChange = (event: React.FormEvent<HTMLInputElement>, index: number) => {
```

Comment:
nit: handleInputChange: <React.FormEventHandler<HTMLInputElement>> = (event) => {} might be more precise.

Reply:
We also have an `index` parameter here.
This doesn't fit the type of `<React.FormEventHandler>`.